client/lib/js/react/stores/asset_store.js

```diff
@@ -64,7 +68,24 @@ class AssetStore {
         }
         this.assetMeta.isFetchingList = false;
     }
-    
+
+    postProcessAsset(asset) {
+        const condition = asset.transaction.conditions[0].condition;
+
+        if (condition.details.hasOwnProperty('subfulfillments') &&
```

Comment:
Probably better to use `Array.isArray(...)` here since it's assumed to be an array at the next check.